src/main/scala/com/fulcrumgenomics/fastq/FastqRecord.scala

```diff
@@ -26,4 +26,20 @@ package com.fulcrumgenomics.fastq
 /**
   * Represents a record that can be read from or written to a fastq file.
   */
-case class FastqRecord(name: String, bases: String, quals: String, comment: Option[String], readNumber: Option[Int])
+case class FastqRecord(name: String, bases: String, quals: String, comment: Option[String], readNumber: Option[Int]) {
+  /** Constructs the header line from the name, read number and comment. */
+  def header: String = name + readNumber.map("/" + _).getOrElse("") + comment.map(" " + _).getOrElse("")
```

Comment:
`readNumber.map("/" + _).getOrElse("") ` could be `readNumber.fold("") { "/" + _ }`; I am not sure which I like better